backend/index.js

```diff
@@ -56,3 +67,14 @@ runCrawlers().catch((error) => {
   console.error(error);
   process.exit(-1);
 });
+
+app.get('/health', (req, res) => {
+  const statuses = status.getAll();
+  const httpStatus = status.isHealthy() ? StatusCodes.OK : StatusCodes.BAD_GATEWAY;
```

Comment:
BAD_GATEWAY is 502 and not 503. Please, fix.

Reply:
fixed